src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -29,7 +29,7 @@
     protected String courseId;
     protected String feedbackSessionName;
     protected FeedbackSubmissionEditPageData data;
-    protected String hasResponses;
+    protected boolean isHasResponse;
```

Comment:
Variable name is weird. Use hasValidResponse since the response has to be valid?